pset-functions.js

```diff
@@ -91,6 +113,24 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
```

Comment:
@Tambini nice job overall, however, be careful with your indentation, it's off near the bottom of the function